dev/generate_bom.py

```diff
@@ -200,6 +200,19 @@ def write_docker_version_files(self):
       with open(config_file, 'w') as cfg:
         cfg.write(docker_tag)
 
+  def write_rpm_version_files(self):
+    """Write a file containing the version number for each microservice. Used
+    by build_release.py for rpm builds.
+    """
+    self.determine_and_tag_halyard()
+    for comp in dict(self.__component_versions.items() + self.__halyard_version.items()):
```

Comment:
If you are just iterating over the keys, why build up the dictionaries as opposed to just the union of their keys?